README.md

```diff
@@ -34,7 +34,6 @@ A notification email will be sent to locked out users.
 stack. This requires to patch `ckan.config.middleware.pylons_app`. The patch is
 currently available in the data.govt.nz [CKAN repository](https://github.com/data-govt-nz/ckan/) on the `dia` branch,
 or [commit `74f78865`](https://github.com/data-govt-nz/ckan/commit/74f78865b8825c91d1dfe6b189228f4b975610a3) for cherry-pick.
-* A running memcached instance and `libmemcached-dev`.
 
```

Comment:
Is it worth mentioning that redis is a requirement?

Is it worth mentioning the limitations around redis, and the consideration of having two redis instances running so that you aren't affected by memory starvation issues on an instance?

Reply:
This was the additional work I had yet to do :)